common/src/main/resources/emails/RejectImageEmail.html

```diff
@@ -0,0 +1,392 @@
+                                    <table style="width: 100%; border-collapse: collapse; border: 0">
+                                        <tr>
+                                            <td style="padding: 10px 30px; font-family: Tahoma, sans-serif">
+                                    <![endif]-->
+                                    <div style="color:#555555;font-family:Montserrat, Trebuchet MS, Lucida Grande, Lucida Sans Unicode, Lucida Sans, Tahoma, sans-serif;line-height:1.5; padding: 10px 30px">
+                                        <div style="line-height: 1.5; font-size: 12px; font-family: Montserrat, Trebuchet MS, Lucida Grande, Lucida Sans Unicode, Lucida Sans, Tahoma, sans-serif; color: #555555; mso-line-height-alt: 18px;">
+                                            <p style="font-size: 15px; line-height: 1.5; text-align: center; word-break: break-word; font-family: inherit; mso-line-height-alt: 23px; margin: 0;"><span style="color: #808389; font-size: 15px;">An admin has rejected your image with the following reason: ${reason}. </span></p>
+                                        </div>
+                                    </div>
+
+                                    <!-- Divider -->
+                                    <table class="divider" border="0" cellpadding="0" cellspacing="0" width="100%" style="table-layout: fixed; vertical-align: top; border-spacing: 0; border-collapse: collapse; mso-table-lspace: 0; mso-table-rspace: 0; min-width: 100%; -ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%;" role="presentation" valign="top">
+                                        <tbody>
+                                        <tr style="vertical-align: top;" valign="top">
+                                            <td class="divider_inner" style="word-break: break-word; vertical-align: top; min-width: 100%; -ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; padding: 60px 0 12px" valign="top">
+                                                <table class="divider_content" border="0" cellpadding="0" cellspacing="0" width="100%" style="table-layout: fixed; vertical-align: top; border-spacing: 0; border-collapse: collapse; mso-table-lspace: 0; mso-table-rspace: 0; border-top: 0 solid #BBBBBB; width: 100%;" align="center" role="presentation" valign="top">
+                                                    <tbody>
+                                                    <tr style="vertical-align: top;" valign="top">
+                                                        <td style="word-break: break-word; vertical-align: top; -ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%;" valign="top"><span></span></td>
+                                                    </tr>
+                                                    </tbody>
+                                                </table>
+                                            </td>
+                                        </tr>a
```

Comment:
⛏️ a (can we get rid of this extra a? it shows up in the template and it looks a little weird)

![image](https://github.com/Code-4-Community/speak-for-the-trees-backend-v2/assets/68914997/73c664c1-1ca1-418a-91a5-1e6cee974117)